From 05472f8798a4dcecb2c3fc1afeca323f1563ef72 Mon Sep 17 00:00:00 2001
From: Balakrishnan Unnithan <balakrishnan.unnithan@intel.com>
Date: Thu, 19 Dec 2024 19:17:37 +0530
Subject: [PATCH 7/7] fix ota update failure cause by vabc config change

Revert PR:
https://github.com/projectceladon/device-androidia-mixins/pull/2173
Set compression method to lz4.

Tracked-On: OAM-128553
Signed-off-by: Balakrishnan Unnithan <balakrishnan.unnithan@intel.com>
---
 target/product/virtual_ab_ota/vabc_features.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/virtual_ab_ota/vabc_features.mk b/target/product/virtual_ab_ota/vabc_features.mk
index 1219763b0d..a13bb87fc3 100644
--- a/target/product/virtual_ab_ota/vabc_features.mk
+++ b/target/product/virtual_ab_ota/vabc_features.mk
@@ -58,7 +58,7 @@ ifndef PRODUCT_VIRTUAL_AB_COMPRESSION_FACTOR
 endif
 
 PRODUCT_VIRTUAL_AB_COMPRESSION := true
-PRODUCT_VIRTUAL_AB_COMPRESSION_METHOD ?= none
+PRODUCT_VIRTUAL_AB_COMPRESSION_METHOD ?= zstd
 PRODUCT_PACKAGES += \
     snapuserd \
 
-- 
2.34.1

